source/manual/create-a-local-transaction.html.md

```diff
@@ -155,45 +156,48 @@ Run [this rake task](https://github.com/alphagov/local-links-manager/blob/a1a2b2
 * Enables the [`Service`](https://github.com/alphagov/local-links-manager/blob/a1a2b23982f62d28ffbe38208807f2d4199548e6/app/models/service.rb#L1) and [`ServiceInteraction`](https://github.com/alphagov/local-links-manager/blob/a1a2b23982f62d28ffbe38208807f2d4199548e6/app/models/service_interaction.rb#L1)
 
 ```bash
-bundle exec rails service:enable[LGSL_CODE,LGIL_CODE,SERVICE_NAME,SERVICE_SLUG]
+$ kubectl -n apps exec local-links-manager-fdc5dbbb7-vrf96 -- bundle exec rails service:enable[LGSL_CODE,LGIL_CODE,SERVICE_NAME,SERVICE_SLUG]
 ```
 
 _Where `SERVICE_NAME` is the service [`name`](#seek-content-advice), and `SERVICE_SLUG` is the service [`slug`](#seek-content-advice)._
 
 Then run [this rake task](https://github.com/alphagov/local-links-manager/blob/a1a2b23982f62d28ffbe38208807f2d4199548e6/lib/tasks/import/missing_links.rake#L5) in Local Links Manager to add (empty) links for all available local authorities...
 
 ```bash
-bundle exec rails import:missing_links
+$ kubectl -n apps exec local-links-manager-fdc5dbbb7-vrf96 -- bundle exec rails import:missing_links
 ```
 
 ## Upload Local Authority URLs to Local Links Manager (LLM)
 
 Upload the completed CSV file that was sent to the product owner [in this step](#get-a-list-of-local-authority-slugs-and-urls)...
 
 ```bash
-$ gds govuk connect scp-push -e integration backend LOCAL_FILE_PATH REMOTE_FILE_PATH
+$ kubectl -n apps cp LOCAL_FILE_PATH local-links-manager-fdc5dbbb7-vrf96:/REMOTE_FILE_PATH
 ```
 
 Run [this rake task](https://github.com/alphagov/local-links-manager/blob/a1a2b23982f62d28ffbe38208807f2d4199548e6/lib/tasks/import/service_links_from_csv.rake#L5) to import the links from the CSV file.
 
 ```bash
-bundle exec rails import:service_links[LGSL_CODE,LGIL_CODE,REMOTE_FILE_PATH]
+$ kubectl -n apps exec local-links-manager-fdc5dbbb7-vrf96 -- bundle exec rails import:service_links[LGSL_CODE,LGIL_CODE,REMOTE_FILE_PATH]
 ```
 
-__Note__: If you intend to run the `import:service_links` rake task via Jenkins, you will need run scp-push against all of the backend machines. For example:
+__Note__: To run the `import:service_links` rake task, you will need to cp against all of the live pods. For example:
```

Comment:
Why tho? 😅 Surely it'd be easier to pick one pod, copy the file to it and run the Rake task on the same pod?